### DISCUSSION TITLE
Obtener_valor_en_posicion TP8

### DISCUSSION THREAD

**Discussion by agusscarava** [INGCOM-UNRN-P1/cursada-2023#205] [Preguntas y respuestas] [answered]
Hola buen domingo a todos !
Queria consultar, que es lo que deberia hacer en esta funcion:

```int obtener_valor_en_posicion(nodo_t* lista, int posicion);```

Gracias de ante mano

**Comment by Nicolino07** [comment, marked as answer]
Segun mi interpretacion deberías obtener el valor "dato" en un nodo determinado,  cada nodo sería una posición . Es decir que deberías comenzar en el nodo  0 y avanzar, nodo 1,2,3... hasta obtener el dato del nodo en la posición X.

**Reply by agusscarava** [reply]
aaaa okok, habia leido mal, es verdad, gracias !

**Reply by martinvilu** [reply]
Hacer lo mismo que harías si fuese una arreglo con `arreglo[3]`
